index.js

```diff
@@ -312,7 +332,8 @@ module.exports = function (schema, options) {
             $unset:{
                 deleted: true,
                 deletedAt: true,
-                deletedBy: true
+                deletedBy: true,
+                deletedId: true
```

Comment:
❓ Is the module setup to support trailing commas? You should probably just do it as we'll probably have to convert it to async/await at some point soon too.